AnnoyancesFilter/sections/popups.txt

```diff
@@ -98,6 +98,8 @@ sodapdf.com##.be-tooltip
 !
 ! Specific sites
 !
+scribd.com#$#.wrapper { display: none !important; }
+scribd.com#$#body { overflow: visible !important; padding-right: 0 !important; }
```

Comment:
`padding-right: 0 !important;` is not requided on my end.

Reply:
Removed.